File: practice.rb
# for i in 1..8
#   p i
# end

# numbers = [1,2,3,4,5,6,7,8]
# numbers.each do |num|
#   puts num
# end

# num = 1
# while num <= 8
#   p num
#   num += 1
# end

# age = 22
# if age >= 20 && age < 30
#   puts "私は20代です"
# end

# pref = "Hiroshima"
# if pref == "Osaka" || pref == "Hiroshima"
#   puts "出身地は大阪か広島です"
# end

class Greeting
  def initialize(name)
    @name = name
  end

  def say_hello
    puts "Hello, #{@name}!"
  end
end

greeting = Greeting.new("Tomoya")
greeting.say_hello

File: practice.py
# for i in range(1, 9):
#     print(i)

# input_value = input("Enter a number! ")
# num = int(input_value)
# if int(num) > 8:
#     print(str(num) + " is bigger than 8")
# elif num < 8:
#     print(str(num) + " is smaller than 8")
# else:
#     print(str(num) + " is equal to 8")


# age = 22
# if age >= 20 and age < 30:
#     print("私は20代です")

# pref = "Hiroshima"
# if pref == "Osaka" or pref == "Hiroshima":
#     print("出身地は大阪か広島です")

class Greeting:
    def __init__(self, name):
        self.name = name

    def say_hello(self):
        print("Hello, " + self + "!")

    name = "Tomoya"
    say_hello(name)
